LDMP/jobs/mvc.py

```diff
@@ -26,6 +33,9 @@
 WidgetDatasetItemUi, _ = uic.loadUiType(
     str(Path(__file__).parents[1] / "gui/WidgetDatasetItem.ui"))
 
+WidgetDatasetItemDetailsUi, _ = uic.loadUiType(
```

Comment:
Please put the details dialog code in its own file. The `jobs.mvc` module is good as it is.